### PR TITLE
New File and correction.

### PR DESCRIPTION
**Pull request by dzpex**

### CODE REVIEW

File: _includes/wiki/it/wikilicense.html [hunk 1]
```diff
@@ -1 +1 @@
-La licenza di questo wiki è soggetta a <a href="https://creativecommons.org/licenses/by-sa/4.0/deed" target="_blank" rel="noreferrer">"Creative Commons Attribution-ShareAlike 4.0 International License"</a>.
+La licenza soggettata a <a href="https://creativecommons.org/licenses/by-sa/4.0/deed" target="_blank" rel="noreferrer">"Creative Commons Attribution-ShareAlike 4.0 International License"</a>.
```
Comment (ann0see) [inline _includes/wiki/it/wikilicense.html:1]: We are currently rethinking the footer. I think @gilgongo removed this file?

**Review by ann0see — CHANGES_REQUESTED** [review]

**Comment by ann0see** [comment]
Thank you again! I'll wait what gilgongo says.

File: _includes/wiki/it/wikilicense.html (hunk 1 above)
Comment (dzpex) [reply _includes/wiki/it/wikilicense.html:1]: I do not see this mods...if you remove its i remove its from my repo...no probl!

Comment (dzpex) [reply _includes/wiki/it/wikilicense.html:1]: i review the "Installation for Windows" file and i see that i make some mistake i need to reload the file and remake the pull request...

**Comment by dzpex** [comment]
ok now you are able to merge the new file

File: _includes/wiki/it/wikilicense.html (hunk 1 above)
Comment (ann0see) [reply _includes/wiki/it/wikilicense.html]: No problem. I can also remove it. The license is now in the footer of the wiki. I'll add it there

**Review by ann0see — APPROVED** [review]

**Pull request merged**